src/plugins/intel_gpu/src/kernel_selector/kernel_selector_utils.h

```diff
@@ -88,7 +88,8 @@ bool UpdateWeightsParams(weight_bias_params& newParams,
                          WeightsReorderParams& weightsReorderParams,
                          const ParamsKey& paramsKey = ParamsKey(),
                          size_t groups = 1,
-                         bool rotate = false);
+                         bool rotate = false,
+                         bool deformable = false);
```

Comment:
Do we need deformable parameter additional? I can't find any deformable is true case.